src/scripts/main.js

```diff
@@ -1736,6 +1780,25 @@
         message.hunt_details = zt;
     }
 
+    /**
+     * Report active augmentations and floor number
+     * @param {Object <string, any>} message The message to be sent.
+     * @param {Object <string, any>} user The user state object, when the hunt was invoked (pre-hunt).
+     * @param {Object <string, any>} user_post The user state object, after the hunt.
+     * @param {Object <string, any>} hunt The journal entry corresponding to the active hunt.
+     */
+    function addValourRiftHuntDetails(message, user, user_post, hunt) {
+        const attrs = user.environment_atts;
+        // active_augmentations is undefined outside of the tower
+        if (attrs.stage === "tower") {
+            message.hunt_details = {
+                super_siphon: attrs.active_augmentations.ss, // 'true' when active, 'undefined' when inactive
+                string_stepping: attrs.active_augmentations.sste, // ^
```

Comment:
Not sure about the backend code - maybe the inactive state should be omitted or converted to `false`

Reply:
`false` is more appropriate
```suggestion
                super_siphon: !!attrs.active_augmentations.ss, // 'true' when active, 'undefined' when inactive
                string_stepping: !!attrs.active_augmentations.sste, // ^
```

Reply:
Good catch